internal/testutils/integ.go

```diff
@@ -0,0 +1,90 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package testutils
```

Comment:
Nit:
I know we don't already do this, but `utils` is discouraged: https://github.com/golang/go/wiki/CodeReviewComments#package-names

Maybe it would be better to call this package `integtest` and rename the method to just `Prepare` so in tests, you get `integtest.Prepare()` Thoughts?

Reply:
I think that is much better than the direction I was heading. Let me push those changes and we can have a second look. :)